docs/en/observability/uptime-set-up-app.asciidoc

```diff
@@ -0,0 +1,168 @@
+[[uptime-set-up-choose-agent]]
+= Use the {uptime-app}
+
+beta[]
+
+You can now create monitors directly in the {uptime-app}.
+To try out this beta functionality, first you will need to enable Monitor Management:
+
+. Go to **{observability}** -> **Uptime**.
+. Click **Monitor Management**.
+. Review the terms and click **Enable** to start using Monitor Management.
+
+From here, you can either <<global-managed-testing-infrastructure,use global managed testing infrastructure>>
+or <<private-locations,create a private location>> to run monitors from your own premises.
+
+NOTE: To use the Elastic Synthetics integration, see the https://www.elastic.co/guide/en/observability/8.3/uptime-set-up.html#uptime-set-up-choose-agent[8.3 documentation]. The Elastic Synthetics integrations is similar to the {uptime-app} approach, but contains fewer configuration options.
```

Comment:
There's a nit here but also I'm a bit concerned about how important it is to mention it has fewer configuration options. Perhaps we should make it more explicit what those configuration differences are or remove it altogether?

```suggestion
NOTE: To use the Elastic Synthetics integration, see the https://www.elastic.co/guide/en/observability/8.3/uptime-set-up.html#uptime-set-up-choose-agent[8.3 documentation]. The Elastic Synthetics integration is similar to the {uptime-app} approach, but contains fewer configuration options.
```

Reply:
@paulb-elastic what do you think?

Reply:
@colleenmcginnis I'm not sure we need to focus on what the different options are, as I think users will question _"which options do I want?"_ to try and decide which one to use.  I think this should make it clear that users should use Private Locations as opposed to the Synthetics Integration (which will be removed in the future).